### ISSUE TITLE
FOIX0001 error?

### ISSUE THREAD

**Issue by apb2006** [GuntherRademacher/markup-blitz#18]
Using BaseX 11.5
```xquery
"https://raw.githubusercontent.com/johnlumley/jwiXML/refs/heads/main/QT4Grammars/xpath40.1.0.ixml"
=>fetch:text()
=>invisible-xml()
```
Generates
```
[FOIX0001] Failed to generate ixml parser: de.bottlecaps.markup.BlitzIxmlException: [S03] 
The grammar contains more than one rule for a given nonterminal name: Expr.
```
Is this correct? I can only see one rule defining `Expr`.

**Comment by GuntherRademacher** [comment]
This is a bug. While transforming to BNF, `-Expr` is reused for a subexpression on the rhs of `SquareArrayConstructor`, and then added a second time, ignoring the fact that it is already there.

Will fix it soon. Thanks for reporting it!

**Comment by GuntherRademacher** [comment]
Fixed in [Markup Blitz 1.5](https://github.com/GuntherRademacher/markup-blitz/releases/tag/1.5) (now on [Maven Central](https://central.sonatype.com/artifact/de.bottlecaps/markup-blitz/1.5))
